Uno.Gallery/Uno.Gallery.UITest/Given_BurgerMenuToggleButton.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using Uno.UITest.Helpers.Queries;
+using Uno.UITests.Helpers;
+
+namespace Uno.Gallery.UITests
+{
+	public class Given_BurgerMenuToggleButton : TestBase
+	{
+		/*This function is to test the Burger Menu Toggle Button */
+		[Test]
+		public void WhenBurgerMenu_01_Click()
```

Comment:
Please leave some comment for the tests and break it in multiple smaller test if needed